evolver/calibration/standard/tests/test_fitting.py

```diff
@@ -4,9 +4,16 @@
 import pytest
 from pydantic import ValidationError
 
+import evolver.settings
+from evolver.base import ConfigDescriptor
 from evolver.calibration.standard.polyfit import LinearCalibrator, LinearTransformer
 
 
+@pytest.fixture()
+def tmp_calibration_dir(monkeypatch, tmp_path):
+    monkeypatch.setattr(evolver.settings.settings, "ROOT_CALIBRATOR_FILE_STORAGE_PATH", tmp_path)
```

Comment:
Semi-unrelated, but I noticed that the tests were creating persisting calibration files.